### PR TITLE
Add commands to documentation

### PR DESCRIPTION
**Pull request by stdevMac**
## Changes:
- Add command description to documentation

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] Documentation Update

### CODE REVIEW

File: docs/docs/quickstart/install-guide.mdx [hunk 1]
```diff
@@ -68,7 +68,7 @@ brew untap nethermindeth/sedge
 
 ## Using Go
 
-First install the Go programming language following the [official instructions](https://go.dev/doc/install). You need at least the `1.18.2` version.
+First install the Go programmning language following the [official instructions](https://go.dev/doc/install). You need at least the `1.18.2` version.
```
Comment (AntiD2ta) [inline docs/docs/quickstart/install-guide.mdx:71]: Why this change?

File: docs/docs/commands/clients.mdx [hunk 2]
```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 1
```
Comment (AntiD2ta) [inline docs/docs/commands/clients.mdx:2]: All the commands have `sidebar_position: 1`, please order them in the order we show them when you run `sedge --help`

**Review by AntiD2ta — CHANGES_REQUESTED** [review]

File: docs/docs/quickstart/install-guide.mdx (hunk 1 above)
Comment (stdevMac) [reply docs/docs/quickstart/install-guide.mdx]: was an error, already resolved

File: docs/docs/commands/clients.mdx (hunk 2 above)
Comment (stdevMac) [reply docs/docs/commands/clients.mdx]: done

**Review by AntiD2ta — APPROVED** [review]
